autogluon/task/tabular_prediction/predictor.py

```diff
@@ -547,6 +551,56 @@ def save(self):
         self._learner.save()
         logger.log(20, "TabularPredictor saved. To load, use: TabularPredictor.load(\"%s\")" % self.output_directory)
 
+    def load_data_internal(self, dataset='train', return_X=True, return_y=True):
+        """
+        Loads the generic internal data representation used during model training.
```

Comment:
unsure what 'generic' means here?  could be confusing for users

Reply:
Good catch, removed.